ground/src/main/res/drawable/ic_vertex.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2021 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="12dp"
+    android:height="12dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M12,12m-10.5,0a10.5,10.5 0,1 1,21 0a10.5,10.5 0,1 1,-21 0"
+      android:strokeWidth="0"
+      android:fillColor="@color/drawAreaLineStringStrokeColor"/>
```

Comment:
Sorry to ask, but we just finalized the designs here:

![image](https://github.com/google/ground-android/assets/228050/6183003e-cc39-4576-a811-19b47a02430f)

Any chance you could update the styling to match? Just asked @rawbzz to confirm whether the color should match the polygon (job color) or be fixed. Thanks for your patience!

Reply:
Yes! No worries, happy to refactor!

Reply:
And I'm assuming that we'll be using the white circle to indicate the vertices, including the endcap, but the plus-sign is going to move independently on its own when panning, to minimize occlusion of the base layer?

Reply:
PTAL!

![Screenshot 2024-02-15 at 5 52 17 PM](https://github.com/google/ground-android/assets/12646706/087aa198-670b-40fd-8ce9-8818f79930ac)
![Screenshot 2024-02-15 at 5 52 30 PM (3)](https://github.com/google/ground-android/assets/12646706/59a158fa-9bbb-40eb-ab88-7587994336f8)
![Screenshot 2024-02-15 at 5 52 49 PM](https://github.com/google/ground-android/assets/12646706/bcbfcf44-e605-4362-8721-deb08e7ab196)

Reply:
> And I'm assuming that we'll be using the white circle to indicate the vertices, including the endcap, but the plus-sign is going to move independently on its own when panning, to minimize occlusion of the base layer?

Hey I hadn't thought of that, but it's a great idea. The only this is that this will only work because of a bug / incomplete FR where the edge only snaps to the map center when the camera stops moving. But the way you've done it makes this a feature rather than a bug, so LGTM. @rawbzz FYI